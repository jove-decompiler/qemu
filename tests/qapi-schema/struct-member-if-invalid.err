tests/qapi-schema/struct-member-if-invalid.json:2: 'if' condition must be a string or a list of strings
