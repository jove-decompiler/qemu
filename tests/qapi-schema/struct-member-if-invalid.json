# Cover member with invalid 'if'
# FIXME not rejected, would generate '#if True\n'
{ 'struct': 'Stru',
  'data': { 'member': { 'type': 'int', 'if': true } } }
