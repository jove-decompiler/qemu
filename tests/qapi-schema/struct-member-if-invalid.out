module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module struct-member-if-invalid.json
object Stru
    member member: int optional=False
        if [True]
